documentation/content/en/books/handbook/introduction/_index.adoc

```diff
@@ -141,13 +141,13 @@ Many of the world's largest IT companies use FreeBSD:
 
 FreeBSD has also spawned a number of related open source projects:
 
-* link:http://bsdrp.net/[BSD Router] - A FreeBSD based replacement for large enterprise routers, designed to run on standard PC hardware.
+* link:http://bsdrp.net/[BSD Router] - A FreeBSD-based replacement for large enterprise routers, designed to run on standard PC hardware.
 * link:https://www.truenas.com/[TrueNAS] is a Network Attached Storage (NAS) software that shares and protects data from modern-day threats like ransomware and malware. TrueNAS makes it easy for users and client devices to access shared data through virtually any sharing protocol.
-* link:https://ghostbsd.org/[GhostBSD] is derived from FreeBSD, uses the GTK environment to provide a beautiful look and comfortable experience on the modern BSD platform offering a natural and native UNIX(R) work environment.
+* link:https://ghostbsd.org/[GhostBSD] is derived from FreeBSD and uses the GTK environment to provide a beautiful look and comfortable experience on the modern BSD platform, offering a natural and native UNIX(R) work environment.
 * link:http://mfsbsd.vx.sk/[mfsBSD] - A toolkit for building a FreeBSD system image that runs entirely from memory.
-* link:https://xigmanas.com/[XigmaNAS] - A file server distribution based on FreeBSD with a PHP powered web interface.
-* link:http://www.opnsense.org/[OPNSense] is an open source, easy-to-use and easy-to-build FreeBSD based firewall and routing platform. OPNsense includes most of the features available in expensive commercial firewalls, and more in many cases. It brings the rich feature set of commercial offerings with the benefits of open and verifiable sources.
-* link:https://www.midnightbsd.org[MidnightBSD] is a FreeBSD derived operating system developed with desktop users in mind. It includes all the software you'd expect for your daily tasks: mail, web browsing, word processing, gaming, and much more.
+* link:https://xigmanas.com/[XigmaNAS] - A file server distribution based on FreeBSD with a PHP-powered web interface.
+* link:http://www.opnsense.org/[OPNSense] is an open source, easy-to-use and easy-to-build FreeBSD-based firewall and routing platform. OPNsense includes most of the features available in expensive commercial firewalls, and more in many cases. It brings the rich feature set of commercial offerings with the benefits of open and verifiable sources.
+* link:https://www.midnightbsd.org[MidnightBSD] is a FreeBSD-derived operating system developed with desktop users in mind. It includes all the software you would expect for your daily tasks: mail, web browsing, word processing, gaming, and much more.
```

Comment:
While here, adjust for documentation style rule that says not to use "you". Something like "software expected for daily tasks".